frontend_tests/zjsunit/namespace.js

```diff
@@ -79,6 +79,9 @@ function load(request, parent, isMain) {
         return obj;
     }
 
+    if (filename.endsWith(".png") || filename.endsWith(".svg")) {
+        return filename;
```

Comment:
This now has no coverage, so needs to be istanbul-ignored or removed until we need it.

Reply:
I dropped the commit that added it, since it's not hard to add back later if needed.